apps/dtsse/dtsse-dashboard-github-ingestion/aat/01.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dtsse-dashboard-github-ingestion
+spec:
+  releaseName: dtsse-dashboard-github-ingestion
+  values:
+    job:
```

Comment:
this should have an image specified with an imagepolicy next to it in the comments so flux can update it

Reply:
I put the image policy in here: https://github.com/hmcts/cnp-flux-config/pull/18026/files#diff-35db3f886a8171ff5b651fb44f5bfb4a0e5fbb4918b0148dccbcc95a9d333dc7R10 - will that fly?